File: Benchmarking_Languages_Project/Julia_Tests/Test5.jl
tic()
for i in 1:15
T5 = reshape(randn(3000*3000),3000,3000)
x = T5[:,2:end]
y = T5[:,1]
(m,n) = size(x)
(q,r) = qr(x)
betas = r[1:n,:]\(transpose(q) * y)
end
timing = toc()
println (timing/15)

File: Benchmarking_Languages_Project/Scala_Tests/Test5.scala
var timing = DenseVector.zeros[Long](15)
for(i <- 0 until 15){
	
	val start = System.currentTimeMillis()
	val x = DenseMatrix.randn(3000,3000)
	val y = DenseMatrix.randn(3000,1)
	val Betas = x\y
	val finish = System.currentTimeMillis()
	timing(i) = (finish - start)
}

println(mean(timing)/1000)

